src/hotspot/share/runtime/java.cpp

```diff
@@ -356,6 +363,8 @@ void print_statistics() {
   }
 
   ThreadsSMRSupport::log_statistics();
+
+  log_vm_init_stats();
```

Comment:
Do we really want to call `ClassLoader::print_counters` here? IIUC most everything else here is printing to tty, but `ClassLoader::print_counters` will "print" to whereever the logging has been configured. (` ThreadsSMRSupport::log_statistics` seems similarly misplaced as it too uses logging).

Reply:
If using `tty`,  the output would lose the logging tag. The output would look as follows:
```
ClassLoader:
  clinit:               11ms / 285 events
  link methods:         13ms / 7493 events
  method adapters:      12ms / 571 events
```
versus with logging tag:
```
[0.094s][info][init] ClassLoader:
[0.094s][info][init]   clinit:               11ms / 278 events
[0.094s][info][init]   link methods:         13ms / 7336 events
[0.094s][info][init]   method adapters:      12ms / 571 events
```

Reply:
Yes I understand that, but this method is generally printing a ton of stuff to the tty - that is what it is for. If we want to add such stuff to the output then it too should just go to the tty - else it doesn't belong in this method IMO.

Reply:
With my new modified fix, the `log_vm_stats` accepts a `outputStream *st` argument so that in `java.cpp` a `tty` would be passed in. In `threads.cpp`, a `LogStreamHandle` could be passed in.